drake/systems/plants/RigidBodyFrame.h

```diff
@@ -3,9 +3,18 @@
 
 class DLLEXPORT_RBM RigidBodyFrame {
 public:
+  RigidBodyFrame(const std::string& _name, const std::shared_ptr<RigidBody>& _body, const Eigen::Matrix4d& _Ttree)
```

Comment:
* What is a Ttree? I know, but it's not going to be clear for anybody else. Use transform_to_body instead?
* why a shared_ptr for body?